pkg/logs/config/integration_config.go

```diff
@@ -57,8 +58,10 @@ type LogsProcessingRule struct {
 type LogsConfig struct {
 	Type string
 
-	Port int    // Network
-	Path string // File
+	Port         int    // Network
+	Path         string // File, Journald
+	IncludeUnits string `mapstructure:"include_units"` // Journald
+	ExcludeUnits string `mapstructure:"exclude_units"` // Journald
```

Comment:
This should be a list a strings, not a string that gets split further down the road.